.tekton/serverless-ingress-pull-request.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/openshift-knative/serverless-operator?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "konflux" && files.all.exists(x, !x.matches('^olm-catalog/|^knative-operator/'))
```

Comment:
I'm not sure if this condition should be same for all images or different.

Reply:
Hmm. I would not do these optiomizations because the operators depend on each other. A simple `grep` gives me this:
* knative-operator depends on openshift-knative-operator
	e.g. pkg/controller/knativekafka/knativekafka_controller.go:	socommon "github.com/openshift-knative/serverless-operator/openshift-knative-operator/pkg/common"

* openshift-knative-operator depends o knative-operator
	e.g. pkg/common/deployment.go:	"github.com/openshift-knative/serverless-operator/knative-operator/pkg/common"

Reply:
Added commit to remove this optimization with explanation in the commit message: https://github.com/openshift-knative/serverless-operator/pull/2707/commits/f72d433b9a24ea788f5869c550a2bbcb905d2cfb